clouddrift/pairs.py

```diff
@@ -0,0 +1,229 @@
+"""
+Functions to analyze pairs of contiguous data segments.
+"""
+from clouddrift import sphere
+import numpy as np
+import pandas as pd
+import xarray as xr
+from typing import Tuple, Union
+
+array_like = Union[list[float], np.ndarray[float], pd.Series, xr.DataArray]
```

Comment:
I understand this being useful as a shortcut but will this be represented in the API/docstring when the user look these up?

Reply:
Not sure yet, will find out.

Reply:
`array_like` defined like this will not appear in parsed docs. Should we be explicit instead, then?